src/abstract-ops/generator-operations.mts

```diff
@@ -33,6 +33,8 @@ export interface GeneratorObject extends OrdinaryObject {
   GeneratorContext: ExecutionContext | null;
   readonly GeneratorBrand: JSStringValue | undefined;
   UnderlyingIterator?: IteratorRecord;
+  // NON-SPEC
+  EnclosedValue?: Value;
```

Comment:
I'm thinking, should we put all captured values in all abstract closures in this field (by adding this field to including BuiltinFunctionObject)

So I'd suggest the name to be `HostCapturedValues?: readonly Value[];`. I'm not clear about the action we should take here (maybe we have more GC issues than this), so I will come back to this PR after I finish the RegExp update.